content/help/ops/security/repairing-citadel/index.md

```diff
@@ -5,6 +5,15 @@ weight: 10
 keywords: [security,citadel,ops]
 ---
 
+Istio currently supports only a **single Citadel instance**. Running multiple Citadel instances may introduce
+race conditions and can **lead to system outages**.
+
+Citadel is not a critical data plane component. The default workload certificate lifetime is 3
+months and certificates will be rotated before they expire, which means if Citadel is down for a
+while, e.g., several hours, existing mutual TLS traffic will not be affected. However,
```

Comment:
Move the however to warning level, and place in front of line 11.

Reply:
done.